src/GitUtils.ps1

```diff
@@ -285,8 +285,13 @@ function Get-GitStatus {
                     $aheadBy = $cacheResponse.AheadBy
                     $behindBy = $cacheResponse.BehindBy
 
-                    if ($cacheResponse.Stashes) { $stashCount = $cacheResponse.Stashes.Length }
-                    if ($cacheResponse.State) { $branch += "|" + $cacheResponse.State }
+                    if ($settings.EnableStashStatus -and $cacheResponse.Stashes) {
```

Comment:
If @cmarcusreid gets bored, the API could conceivably receive request options like this to prevent the cache from doing background work the user doesn't care.